homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
```

Comment:
```py
    OverkizSensorDescription(
        key=OverkizState.CORE_SENSOR_DEFECT,
        name="Sensor defect",
        entity_registry_enabled_default=False,
        entity_category=EntityCategory.DIAGNOSTIC,
        translation_key="sensor_defect",
        native_value=lambda value: OVERKIZ_STATE_TO_TRANSLATION.get(
            cast(str, value), cast(str, value)
        ),
    ),
```
What do these respond? Should it be an enum?

Reply:
https://github.com/home-assistant/core/pull/143241